### ISSUE TITLE
UI for popup notifications

### ISSUE THREAD

**Issue by nareddyt** [nareddyt/discover-rewards-notifier#72]
We need a UI for our **popup notifications** milestone. For example, AdGuard shows this popup whenever it detects blocked ads on a page:

![image](https://user-images.githubusercontent.com/11142171/39668519-ba8069f6-5084-11e8-8ada-1783fd0ad242.png)

We would need something similar to this for the milestone. It should display everything that the main UI currently does (offer name, site name, site picture, link to offer, expiry date, etc). It should also have the following features:

- A button to close the popup
- A drop-down menu to hide popup notifications using specified rules. A user should be able to select between hiding all popups for **a page**, **a specific deal / cashback reward**, for **a set time interval** (like 1 hour), or to disable the feature completely!

**Comment by DerrickTheodore** [comment]
Hey Tejasvi, I want to start working on this.  Anything I should know?  If

**Comment by nareddyt** [comment]
Nope, I think that is it! If you wanna make some UI mockups and post them here for feedback, feel free to. Otherwise you can just go ahead and create the new UI. Please try to stick with the theme of the extension's current UI :)

**Comment by dwouca** [comment]
@DerrickTheodore Hey, by any chance are you still working on this?
